docs/api.md

```diff
@@ -32,6 +32,18 @@ Finally, **system information and configuration** is available via a set of spec
 
 Here major and breaking changes to the API are listed by version.
 
+### ODK Central v1.5
+
+ODK Central v1.5 adds editable Project descriptions as well as more detailed information about Forms and Submissions when listing Projects (via the API and shown in new Project and Form lists on Frontend).
```

Comment:
"Frontend" is more of an internal name. In the API docs, we use a few different terms for Frontend, for example, "the administration website". But since these are the API docs, it might not be necessary to mention Frontend here.